### PR TITLE
Fix build, add packages, expand README, GitHub actions

### PR DESCRIPTION
**Pull request by siraben**

### CODE REVIEW

File: rM/oecore-raw.nix [hunk 1]
```diff
@@ -1,20 +1,34 @@
...
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "rm-toolchain-bin";
   version = "1.8-23.9.2019";
-  src = import <nix/fetchurl.nix> {
-    url = "https://remarkable.engineering/oecore-x86_64-cortexa9hf-neon-toolchain-zero-gravitas-1.8-23.9.2019.sh";
-    sha256 = "cbd5ffa0e2ee4b244d5780289d3950f6228d8e3372f90c0dd728b45201ca61e6";
+
+  src = fetchurl {
+    url = "https://remarkable.engineering/oecore-x86_64-cortexa9hf-neon-toolchain-zero-gravitas-${version}.sh";
```
Comment (SuperSandro2000) [inline rM/oecore-raw.nix]: I think you should mark this package x86_64 only.

Comment (siraben) [reply rM/oecore-raw.nix]: Thanks!

**Comment by siraben** [comment]
Building `rmPkgs.rM-vnc-server` currently fails, see [build](https://travis-ci.com/github/siraben/nix-remarkable/jobs/422045318).

However [rmPkgs.linuxPackages.mxc_epdc_fb_damage](https://travis-ci.com/github/siraben/nix-remarkable/jobs/422045317) and [hostPkgs.gst-libvncclient-rfbsrc](https://travis-ci.com/github/siraben/nix-remarkable/jobs/422045319) build fine.

**Comment by siraben** [comment]
Bisecting, looks like https://github.com/NixOS/nixpkgs/commit/3a57356721f44b8369c09e54ccad97857fa593d4 is the bad commit.

<details>

```
git bisect start
# good: [bf60834d6dfa41a87f357f48f74a56b718860be8] Merge pull request #87748 from Gaelan/nss-arm-fix
git bisect good bf60834d6dfa41a87f357f48f74a56b718860be8
# bad: [b52da4e64108eb1496c037b4ca0f347624e50d51] Merge pull request #102071 from zakame/contrib/perl-NetAsyncWebSocket
git bisect bad b52da4e64108eb1496c037b4ca0f347624e50d51
# bad: [a3c71c3e5dd688fc07628d360e48da0be23e74d7] fuse3: 3.9.2 -> 3.9.4 (#95072)
git bisect bad a3c71c3e5dd688fc07628d360e48da0be23e74d7
# good: [00aebe90110f0c8e8d9381b2f02c307df36dc94d] Merge pull request #89204 from andersk/musescore
git bisect good 00aebe90110f0c8e8d9381b2f02c307df36dc94d
# good: [e99389a94238c5c9086e6bf357a8969ca5ac48f5] Merge pull request #93001 from aanderse/gitolite
git bisect good e99389a94238c5c9086e6bf357a8969ca5ac48f5
# good: [06a0f2aeb6122dc74ce8ae0f772e03cbdd48c7c7] linux: 4.19.134 -> 4.19.135
git bisect good 06a0f2aeb6122dc74ce8ae0f772e03cbdd48c7c7
# good: [87e39d444c34d9f9c392a99fc6313729dae3fde3] Merge pull request #94634 from zowoq/delta
git bisect good 87e39d444c34d9f9c392a99fc6313729dae3fde3
# bad: [8bb50e72d690362e6b5c6d504bcb392e6eba12b7] Allow jdiskreport to build on Darwin
git bisect bad 8bb50e72d690362e6b5c6d504bcb392e6eba12b7
# bad: [be04d5337ebe938282c98d3685c69ec61ae91395] alot: Use bundled theme files
git bisect bad be04d5337ebe938282c98d3685c69ec61ae91395
# bad: [c2366ae35036be6e1b26ec36ad6c9cf4d346ed27] messagelib: no longer depend on qtwebkit
git bisect bad c2366ae35036be6e1b26ec36ad6c9cf4d346ed27
# bad: [b52da4e64108eb1496c037b4ca0f347624e50d51] Merge pull request #102071 from zakame/contrib/perl-NetAsyncWebSocket
git bisect bad b52da4e64108eb1496c037b4ca0f347624e50d51
# bad: [535a3e8d48e1a390bb7097fc7995f2d628855d40] mesa: add patch to link radv with build-id
git bisect bad 535a3e8d48e1a390bb7097fc7995f2d628855d40
# good: [2988feba8cbcf87cc1156f924ad38d5918215a4c] Merge branch 'master' into staging-next
git bisect good 2988feba8cbcf87cc1156f924ad38d5918215a4c
# good: [5bd8316b6e115b5261369e9aa0e88a78a13cc950] Merge #91975: Misc static cross fixes (into staging)
git bisect good 5bd8316b6e115b5261369e9aa0e88a78a13cc950
# good: [331b6429d481cc315a1215b9e60255004d8832ad] python3Packages.cirq: 0.8.0 -> 0.8.2
git bisect good 331b6429d481cc315a1215b9e60255004d8832ad
# good: [d9da3e357b74a4a604ad665568cf6e599bccb2d9] Merge pull request #91446 from prusnak/protobuf-latest
git bisect good d9da3e357b74a4a604ad665568cf6e599bccb2d9
# bad: [2d6b1f123972c4dabc5bf4aa50af1b3402823f03] libuv: disable flaky test "udp_multicast_join" on darwin
git bisect bad 2d6b1f123972c4dabc5bf4aa50af1b3402823f03
# bad: [72d4d15d7799c13a7fbb8d53ade9dfdc98a1f353] python3Packages.pybind11: 2.4.3 -> 2.5.0
git bisect bad 72d4d15d7799c13a7fbb8d53ade9dfdc98a1f353
# bad: [3a57356721f44b8369c09e54ccad97857fa593d4] cmake: 3.17.3 -> 3.18.0
git bisect bad 3a57356721f44b8369c09e54ccad97857fa593d4
# first bad commit: [3a57356721f44b8369c09e54ccad97857fa593d4] cmake: 3.17.3 -> 3.18.0
```
</details>

**Comment by roosemberth** [comment]
Why was this MR closed?

**Comment by roosemberth** [comment]
@siraben ^^

**Comment by siraben** [comment]
@roosemberth Oops! I renamed `fix-build` to `develop` and it closed the PR!

**Comment by siraben** [comment]
Closed since diverged a lot from branch

**Pull request closed without merging**